src/runner/runESLint.js

```diff
@@ -1,7 +1,94 @@
-const { pass, fail, skip } = require('create-jest-runner');
 const { CLIEngine, ESLint } = require('eslint');
 const getESLintOptions = require('../utils/getESLintOptions');
 
+/*
+ * This function exists because there are issues with the `pass`, `skip`, and
+ * `fail` functions from `create-jest-runner`:
+ *
+ * 1. The `pass` and `skip` functions have a bug in our version of
+ *    `create-jest-runner` where calling them will actually pass an `undefined`
+ *    item in `testResults`, which causes the GithubActionsReporter to report an
+ *    empty error message on every file.  This has been resolved in later
+ *    versions of `create-jest-runner`, but upgrading is a breaking change that
+ *    is incompatible with some node versions we support.
+ *
+ * 2. The `fail` function in `create-jest-runner` does not support passing
+ *    multiple failure messages, which makes it impossible to annotate each
+ *    eslint failure.  This has not been resolved, although presumably could be
+ *    worked around by using the underlying `toTestResult` function instead
+ *    (although that function isn't exposed in the public API of that library).
+ *
+ * TODO At some point, we should put a PR in to `create-jest-runner` to resolve
+ * point 2 above, and then should upgrade and remove this function and go back
+ * to using `pass`, `skip`, and `fail` from that library instead.
```

Comment:
are you up for sending that PR?

Reply:
Yeah, I can do that.  I just realized you own both of these repos -- do you have a preference for me to do that before / after this lands?

Reply:
also, can you advise on if you want `toTestResult` to get exposed vs. just adding support for multiple pass/fail/skip/todo messages?

Reply:
> do you have a preference for me to do that before / after this lands?

No preference on timing - this repo is stuck on an older version for now anyways.

> also, can you advise on if you want `toTestResult` to get exposed vs. just adding support for multiple pass/fail/skip/todo messages?

happy to expose the underlying helper as well

Reply:
@cprussin came over this again now - would still love to see that PR for `fail` 😀

Reply:
Hmm jog my memory, I left the following comment:

> This has been resolved in later versions of `create-jest-runner`, but upgrading is a breaking change that is incompatible with some node versions we support.

Is that still accurate?  If so, even if I put that PR in for `fail`, it won't change anything here since we still can't upgrade right?

Reply:
That is correct 👍